kubedifflib/tests/test_diff.py

```diff
@@ -95,3 +96,58 @@ def test_two_lists_of_same_size_generator(items):
     """two_lists_of_same_size returns two lists of the same length."""
     xs, ys = items
     assert len(xs) == len(ys)
+
+
+def kube_spec():
+    """
+    Generate a random kubernetes spec
+    """
+    return fixed_dictionaries({
+        "name": text()
+    })
+
+
+def kube_obj():
+    """
+    Generate a random Kubernetes object
+    """
+    return fixed_dictionaries({
+        "apiVersion": sampled_from(["v1", "v1beta1", "extensions/v1"]),
+        "kind": sampled_from(["Namespace", "Pod", "Job", "CronJob", "Deployment"]),
+        "metadata": fixed_dictionaries({
+            "name": text()
+        }),
+        "spec": one_of(none(), kube_spec())
+    })
+
+
+def kube_list():
+    """
+    Generate a Kubernetes List type with a set of objects
+    """
+    return fixed_dictionaries({
+        "apiVersion": sampled_from(["v1", "v1beta1"]),
+        "kind": sampled_from(["List"]),
+        "items": lists(kube_obj(), min_size=1)
+    })
+
+
+@given(data=kube_list())
+def test_from_dict_kubernetes_list_type(data):
+    """
+    KubeObject.from_dict parses kubernetes lists correctly, by returning each of the
+    items in the list.
```

Comment:
Thanks so much for writing these tests, and for providing docstrings for them too. I'm content with these being merged as-is, but if you want to make the docstrings better, https://jml.io/test-docstrings has some potentially useful guidance.

The relevant bit is that the word "correctly" is redundant, so it's better to try to describe the actual behaviour being tested. Here, you could just elide the word "correctly, " and everything would be 💯